### PR TITLE
Add gradle plugin for google java format. Enforce all files to be formatted.

### PR DESCRIPTION
**Pull request by bogdandrutu**
To format all files:
./gradlew goJF

To verify that all files are correctly formatted:
./gradlew verGJF

Added a travis check for all files to be well formatted.

### CODE REVIEW

File: scripts/travis_script [hunk 1]
```diff
@@ -20,6 +20,7 @@ case "$TASK" in
             export JAVA_HOMES="$(jdk_switcher home openjdk6)/jre:$(jdk_switcher home openjdk7)/jre:$(jdk_switcher home oraclejdk8)/jre:$(jdk_switcher home oraclejdk9)"
             ./gradlew clean assemble --stacktrace
             ./gradlew check :opencensus-all:jacocoTestReport
+            ./gradlew verGJF
```
Comment (sebright) [inline scripts/travis_script]: Could you please add a note about the new check and gradle commands to CONTRIBUTING.md?

**Comment by ubschmidt2** [comment]
I'll have a few conflicts to resolve, but should be ok.

File: api/src/main/java/io/opencensus/stats/Measure.java [hunk 2]
```diff
@@ -46,35 +44,29 @@
...
-   * <p>The suggested grammar for a unit is as follows:
-   * Expression = Component { "." Component } { "/" Component } ;
-   * Component = [ PREFIX ] UNIT [ Annotation ] | Annotation | "1" ;
-   * Annotation = "{" NAME "}" ;
-   * For example, string “MBy{transmitted}/ms” stands for megabytes per milliseconds, and the
-   * annotation transmitted inside {} is just a comment of the unit.
+   * <p>The suggested grammar for a unit is as follows: Expression = Component { "." Component } {
+   * "/" Component } ; Component = [ PREFIX ] UNIT [ Annotation ] | Annotation | "1" ; Annotation =
+   * "{" NAME "}" ; For example, string “MBy{transmitted}/ms” stands for megabytes per milliseconds,
+   * and the annotation transmitted inside {} is just a comment of the unit.
```
[the remark this answers is not in the document]
Comment (bogdandrutu) [reply api/src/main/java/io/opencensus/stats/Measure.java]: Done.

File: scripts/travis_script (hunk 1 above)
Comment (bogdandrutu) [reply scripts/travis_script]: Filed an issue https://github.com/census-instrumentation/opencensus-java/issues/773

**Comment by adriancole** [comment]
Yay!

**Pull request merged**